### PR TITLE
Move internal/library to internal, xerox font dirs, loadup and medleydir

### PR DESCRIPTION
**Pull request by masinter**

### CODE REVIEW

**Comment by nbriggs** [comment]
Loadups fail because internal/library doesn't exist and is referenced when building the whereis database.

**Comment by nbriggs** [comment]
README.md,  internal/MEDLEY-UTILS, lispusers/MEDLEY-FIX-DIRS.LCOM all still use/mention `internal/library`.

**Comment by masinter** [comment]
MEDLEY-UTILs change got lost in the destructuring.
MEDLEY-FIX-DIRS was left over (part of internal/MEDLEY-UTILS).
README.md use was in a list of examples, updated it a little

File: README.md [hunk 1]
```diff
@@ -129,15 +124,15 @@ Each directory should have a README.md, but briefly
 * clos -- early implementation of Common Lisp Object System
 * CLTL2 -- files submitted to bring Medley up to the conformance to "Common Lisp, the Language" 2nd edition. Not enough to conform to the ANSII standard lisp.
 * Dockerfile -- used when building Docker containers with Medley
-* docs -- Documentation files (either PDFs or online help; see medley/wiki)
+* docs -- Documentation files (in TEdit format PDFs or online help; look [here](https://github.com/Interlisp/medley/Documentation)
 * fonts -- raster fonts (or font widths) in various resolutions for display, postscript, interpress, press formats
 * greetfiles -- various configuration setups
-* internal -- These _were_ internal to Venue; now internal/library and internal/test
+* internal -- These _were_ internal to Venu
```
Comment (nbriggs) [inline README.md:130]: Venue

**Review by nbriggs — COMMENTED** [review]
Other than the Venue typo, looks good to me.

**Review by nbriggs — APPROVED** [review]
Looks good.  Loadups work, resulting sysout works.

**Pull request merged**